### PR TITLE
Bravenewcoin snapshot

### PR DESCRIPTION
**Pull request by austinborn**
## Description
Add snapshot integration testing for BraveNewCoin EA

## Changes
- Add snapshot test for successful `crypto` response
- Add snapshot test for successful `vwap` response

## Steps to Test

`yarn && yarn setup && yarn test bravenewcoin/test/integration`

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

**Comment by austinborn** [comment]
[feature/sc-18337]

File: packages/sources/bravenewcoin/test/integration/cryptoFixtures.ts [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        '100',
+        'ETag',
+        'W/"64-3kXL7+uvKLguB6cSInOvQtu4RgM"',
+        'Date',
+        'Tue, 19 Oct 2021 20:46:48 GMT',
+        'Connection',
+        'close',
+      ],
+    )
+}
```
Comment (boxhock) [inline packages/sources/bravenewcoin/test/integration/cryptoFixtures.ts:189]: Can't mock the adapter we're testing :wink: 

```suggestion
```

File: packages/sources/bravenewcoin/test/integration/adapter.test.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    describe('bravenewcoin replies with success', () => {
+      it('should reply with success', async () => {
+        mockAuthTokenResponse()
+        mockBtcCoinEndpoint()
+        mockBtcMarketEndpoint()
+        mockCryptoResponse()
+        mockEthCoinEndpoint()
+        mockEthMarketEndpoint()
```
[the remark this answers is not in the document]
Comment (austinborn) [reply packages/sources/bravenewcoin/test/integration/adapter.test.ts:61]: Yeah I like the idea of bundling endpoints when possible. It's a bit hard to read as-is.

**Review by boxhock — APPROVED** [review]
Looks good :+1:

**Pull request merged**